bob-apps/bin/blob_recovery_tool/command.rs

```diff
@@ -248,13 +251,106 @@ impl ValidateIndexCommand {
     }
 }
 
+pub struct MigrateCommand {
+    input_path: PathBuf,
+    output_path: PathBuf,
+    blob_suffix: String,
+    validate_every: usize,
+    target_version: u32,
+}
+
+impl MigrateCommand {
+    fn run(&self) -> AnyResult<()> {
+        process_files_recursive(
```

Comment:
It would be nice if `migrate` command can handle a single file. If `input_path` is a directory then the command recursively processes all BLOBs, if it is a specific file then it only processes that single file

Reply:
Actually, the same behaviour would be good also for `validate-blob` and `validate-index`

Reply:
Done